services/app-web/src/tealium/tealium.ts

```diff
@@ -7,203 +7,12 @@ import {getRouteName} from '../routeHelpers';
-    event_name: 'filters_applied'
-    search_result_count: string
-    link_type?: 'link_other'
-    results_count_after_filtering: string
-    results_count_prior_to_filtering: string
-    filter_categories_used: string
-}
-
-type TealiumFilterRemovedType = {
-    event_name: 'filter_removed'
-    search_result_count: string
-    link_type?: 'link_other'
-    filter_categories_used: string
-}
-
-type TealiumInlineErrorObject = {
-    event_name: 'inline_error'
-    error_type: 'validation' | 'system'
-    error_message: string
-    error_code?: string
-    form_field_label: string
-    link_type?: 'link_other'
-}
-
-type TealiumAlertImpressionObject = {
-    event_name: 'alert_impression'
-    error_type: 'validation' | 'system'
-    error_message: string
-    error_code?: string
-    heading: string
-    type: 'alert' | 'warn' | 'error'
-    extension?: string
-}
-
-type TealiumRadioButtonEventObject = {
-    event_name: 'radio_button_list_selected'
-    radio_button_title: string
-    list_position: number
-    list_options: number
-    link_type?: 'link_other'
-    parent_component_heading?: string
-    parent_component_type?: string
-    field_type: 'optional' | 'required'
-    form_fill_status: boolean
-}
-
-type TealiumCheckboxEventObject = {
-    event_name: 'checkbox_selected' | 'checkbox_unselected'
-    text: string
-    heading: string
-    parent_component_heading?: string
-    parent_component_type?: string
-    field_type: 'optional' | 'required'
-}
-
-type TealiumFilterEventObject = (
-    TealiumFilterAppliedType | TealiumFilterRemovedType
-)
-
-type TealiumLinkDataObject = {
-    tealium_event: TealiumEvent // event is required for user tracking links
-} & Partial<TealiumDataObject>
-
-type TealiumViewDataObject = TealiumDataObject // event default to page_view in useTealium hook
-
-type TealiumEventObjectTypes = (
-    | TealiumButtonEventObject
-    | TealiumLinkEventObject
-    | TealiumDropdownSelectionEventObject
-    | TealiumFilterEventObject
-    | TealiumInlineErrorObject
-    | TealiumAlertImpressionObject
-    | TealiumCheckboxEventObject
-    | TealiumRadioButtonEventObject
-    ) & Partial<TealiumDataObject>
-
-type TealiumClientType = {
-    initializeTealium: () => void
-    logUserEvent: (
-        linkData: TealiumEventObjectTypes,
-        pathname: string,
-        loggedInUser?: User,
-        heading?: string | React.ReactElement
-    ) => void
-    logPageView: (
-        pathname: string,
-        loggedInUser?: User,
-        heading?: string | React.ReactElement
-    ) => void
-}
+import { TEALIUM_CONTENT_TYPE_BY_ROUTE } from './constants';
+import {removeNullAndUndefined} from'../common-code/data/dataUtilities'
+import { TealiumClientType, TealiumEnv, TealiumEventObjectTypes, TealiumInteractionEventDataObject, TealiumViewDataObject } from './types';
 
 const tealiumClient = (tealiumEnv: Omit<TealiumEnv, 'dev'>): TealiumClientType => {
-    const isProd = tealiumEnv == 'prod'
```

Comment:
Removing this `isProd` logic is what will turn on analytics again on production. They have been incrementally migrating environments and are ready for our prod code to be turned on.